content/features/proxy2/install.md

```diff
@@ -0,0 +1,108 @@
++++
+title = 'Install the built-in proxy'
+weight = 20
++++
+
+The Memcached built-in proxy is available only if you activate a specific configuration option while compiling Memcached. For this reason, you can run the proxy only if you build Memcached from source. This page guides you through through the following tasks:
+
+* Build and install Memcached with its proxy features enabled.
+* Download the Lua library that the proxy requires for common tasks.
+
+For a general overview of the proxy, see [Proxy]({{<legacy_proxy_base_path>}}). For a hands-on tutorial that guides you through compiling and running the proxy with a simple configuration, see [Built-in Proxy Quickstart]({{<proxy_base_path>}}quickstart).
+
+## Before you begin {#prereqs}
+
+Make sure that your machine has the necessary development tools and libraries
+to build Memcached from source. These requirements are probably available from your operating system's package manager.
+
+For example, on Debian, you can satisfy these requirements by having the following packages installed:
+
+* `gcc`
+* `make`
+* `libevent-dev`
+
+The steps on this page guide you though launching three Memcached servers on TCP ports `11211`, `11212`, and `11213`. If you already have Memcached services running on those ports and you don't want to disrupt them, then you should take either of these additional steps:
+
+* Substitute different port numbers for this tutorial.
+* Run this tutorial as written on a different machine that isn't running Memcached.
+
+## Download and build the proxy {#download}
+
+To download and compile Memcached with its proxy features enabled, follow these steps:
+
+1. Download the latest Memcached release tarball:
+
+    ```console
+    wget https://memcached.org/latest
+    ```
+
+1. Unzip and untar the tarball, then change your working directory to the source directory:
+
+    ```console
+    tar xzvf memcached-{{<var>}}VERSION{{</var>}}.tar.gz
+
+    cd memcached-{{<var>}}VERSION{{</var>}}
+    ```
+    
+    Replace <var>`VERSION`</var> with the version number in the tarball that you downloaded in the previous step.
+    
+1. Configure the Memcached build process to enable the built-in proxy:
+
+    ```console
+    ./configure --with-proxy {{<var>}}OTHER_FLAGS{{</var>}}
+    ```
+    
+    Replace <var>OTHER_FLAGS</var> with a list of any other other Memcached configuration options that you wish to enable. For a complete list of these options, run the command `./configure --help`.
```

Comment:
Should probably double check this, but the when referring to options to `configure` people typically refer to them as `configure options`, or more simply compilation options.

Not sure how to check this now that I bring it up... `configure` is a specific thing (autotools) and referring to it as "configure options" might be too generic... but a lot of people don't know what autotools even are anymore...

Reply:
I've renamed the placeholder to OTHER_OPTIONS and rephrased the text to refer to them as "other options for the `configure` command".